Cargo.toml

```diff
@@ -21,7 +21,7 @@ features = ["extension-module"]
 [dependencies]
 itertools = "0.8"
 failure = "0.1"
-finalfusion = { git = "https://github.com/finalfusion/finalfusion-rust.git", rev = "8e360c6" }
+finalfusion = { git = "https://github.com/finalfusion/finalfusion-rust.git", rev = "a7fec83" }
```

Comment:
Do we still need the git version, now that 0.8 is released?

Reply:
Nope, this can be updates to 0.8.